software/bwameth/index/main.nf

```diff
@@ -0,0 +1,35 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+def options    = initOptions(params.options)
+
+process BWAMETH_INDEX {
+    tag "$fasta"
+    label 'process_high'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:'') }
+
+    conda (params.enable_conda ? "bioconda::bwameth=0.2.2" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/bwameth:0.20--py35_0"
+    } else {
+        container "quay.io/biocontainers/bwameth:0.20--py35_0"
+    }
+
+    input:
+    path fasta, stageAs: "index/genome.fa"
+
+    output:
+    path "index"        , emit: index
```

Comment:
```suggestion
    path "bwameth"        , emit: index
```

Reply:
Given the way we are creating this for other indexing modules be great if we can be consistent. See [here](https://github.com/nf-core/modules/blob/01c0d335ca57b1b9298a7acd643116917953804a/software/bwa/index/main.nf#L25)